proxy/airdropper/airdropper.py

```diff
@@ -199,6 +236,90 @@ def airdrop_to(self, eth_address, airdrop_galans):
+            LOG.debug(f"{sol_sig} instruction: {instruction} {sol_neon_ix.neon_tx_sig}")
+            if instruction == EVM_LOADER_HOLDER_WRITE:
+                neon_tx_id = NeonIndexedHolderInfo.Key(sol_neon_ix.get_account(0), sol_neon_ix.neon_tx_sig)
+                data = sol_neon_ix.ix_data[41:]
+                chunk = NeonIndexedHolderInfo.DataChunk(
+                    offset=int.from_bytes(sol_neon_ix.ix_data[33:41],'little'),
+                    length=len(data),
+                    data=data
+                )
+                neon_tx_data = self.neon_large_tx.get(neon_tx_id.value, None)
+                if neon_tx_data is None:
+                    LOG.debug(f"{sol_sig} New NEON trx: {neon_tx_id} {len(chunk.data)} bytes at {chunk.offset}")
+                    neon_tx_data = NeonIndexedHolderInfo(neon_tx_id)
+                    self.neon_large_tx[neon_tx_id.value] = neon_tx_data
+                neon_tx_data.add_data_chunk(chunk)
+                neon_tx_data.add_sol_neon_ix(sol_neon_ix)
+
+            elif instruction in [EVM_LOADER_TRX_STEP_FROM_ACCOUNT,
+                                 EVM_LOADER_TRX_STEP_FROM_ACCOUNT_NO_CHAINID,
+                                 EVM_LOADER_TRX_EXECUTE_FROM_ACCOUNT]:
+                # Do not process iterations without receipt (until we don't need logs we can skip it)
+                if sol_neon_ix.neon_tx_return is None:
+                    continue
+
+                neon_tx_id = NeonIndexedHolderInfo.Key(sol_neon_ix.get_account(0), sol_neon_ix.neon_tx_sig)
+                neon_tx_data = self.neon_large_tx.get(neon_tx_id.value, None)
+                if neon_tx_data is None:
+                    LOG.warning(f"{sol_sig} Holder account {neon_tx_id} is not in the collected data")
+                    continue
+
+                LOG.debug(f"{sol_sig} Finalize {neon_tx_id}")
+                if sol_neon_ix.neon_tx_return.status == 1:
+                    data_hash = '0x'+keccak_256(neon_tx_data.data).hexdigest()
+                    if data_hash != sol_neon_ix.neon_tx_sig:
+                        LOG.warning(f"{sol_sig} Data hash {data_hash} does not match transaction hash {sol_neon_ix.neon_tx_sig}")
+                    else:
+                        self.process_neon_transaction(sol_neon_ix, neon_tx_data.data)
+                self.neon_large_tx.pop(neon_tx_id, None)
+
+            elif instruction == EVM_LOADER_CALL_FROM_RAW_TRX:
+                message = sol_neon_ix.ix_data[5:]
+                self.process_neon_transaction(sol_neon_ix, message)
+
+            elif instruction == EVM_LOADER_STEP_FROM_RAW_TRX:
+                message = sol_neon_ix.ix_data[13:]
+                self.process_neon_transaction(sol_neon_ix, message)
+
+            elif instruction == EVM_LOADER_CANCEL:
+                neon_tx_id = NeonIndexedHolderInfo.Key(sol_neon_ix.get_account(0), sol_neon_ix.neon_tx_sig)
+                self.neon_large_tx.pop(neon_tx_id, None)
```

Comment:
`self.neon_large_tx.pop(neon_tx_id.value, None)` ?

Reply:
And here. Fixed.